test_junit_xml.py

```diff
@@ -73,8 +73,8 @@ def test_single_suite_no_test_cases(self):
 
         (ts, tcs) = serialize_and_read(
             TestSuite(
-                'test',
-                [],
+                name='test',
```

Comment:
Off course not. My comment was about this: imagine you have a function:
```
def foo(a, b, c): ...

def test_foo():
     foo('1', '2', [])
```
If you change the signature of `foo` to `foo(a, c, b)` you break the client code and the test as well. And I thought that realised and fixed that problem just modifying the test:
```
def test_foo():
     foo(a='1', b='2', c=[])
```
So the test now works but still there are guys out there whose code broke :) So the failing test was a red flag here. If you did not - then it's fine.